hyperspy/io_plugins/msa.py

```diff
@@ -263,6 +262,13 @@ def parse_msa_string(string, filename=None):
     else:
         # Defaulting to EELS looks reasonable
         mapped.set_item('Signal.signal_type', 'EELS')
+    if 'YUNITS' in parameters.keys():
```

Comment:
Sorry, that comment wasn't very clear: I was proposing to do that when YLABEL and/or YUNITS are not defined.